actionners/kubernetes/terminate/terminate.go

```diff
@@ -50,6 +50,7 @@ rules:
     grace_period_seconds: 5
     ignore_daemonsets: true
     ignore_statefulsets: true
+	ignore_pods: false
```

Comment:
```suggestion
  ignore_pods: false
```

for consistency, set to "true" all settings, I prefer `delete_standalone_pods`. It also mean, the default value is `false` like it is right now

Reply:
I started with delete_standalone_pods, but for consistency I decided to change it to ignore_pods.

I can set it to true, but are you sure changing it here has any effect on the real app your you're meaning more from the documentation perspective?

Reply:
Changed it to true.